### ISSUE TITLE
Accent char makes keyword_end error

### ISSUE THREAD

**Issue by wbotelhos** [rubocop-hq/rubocop#5570]
```
def foo
  'é'
end
```

## Expected behavior

No error message

## Actual behavior

It show the following error: "unexpected end-of-input, expecting keyword_end"

## Steps to reproduce the problem

Just put an accented character inside quotes

## RuboCop version

```
0.52.1 (using Parser 2.4.0.2, running on ruby 2.4.3 x86_64-darwin17)
```

**Comment by pocke** [comment]
> It show the following error: "unexpected end-of-input, expecting keyword_end"

It is a syntax error. Can you check the full output? Probably it outputs "Lint/Syntax".

However, I cannot reproduce the issue.
And it looks a valid ruby code. Because it passes `ruby -c` and `ruby-parse`.

```bash
$ cat test.rb
def foo
  'é'
end

$ ruby -c test.rb
Syntax OK

$ ruby-parse test.rb
(def :foo
  (args)
  (str "é"))
```

So, I guess it is an encoding issue. Can you check the encoding? You can use `file` command (e.g. `file test.rb`). And can you check `LANG` environment with `echo $LANG`?

**Comment by wbotelhos** [comment]
The error shows on `test.rb` and the syntax is ok:

```
$ ruby -c test.rb
Syntax OK
```

Looks like my $LANG is empty:

```
$ echo $LANG

```

**Comment by ryanhageman** [comment]
Is this still an issue?

**Comment by wbotelhos** [comment]
For now we can close it.
It happens inside my Atom editor, but the plugin <https://atom.io/packages/linter-rubocop> is not work anymore, since I cannot config the path.

Thank you all.
